src/components/Voting/CongressionalAssemble/CongressionalCountdown/index.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import { formatDistance, format } from 'date-fns';
+import congressionalCountdown from './styles.module.scss';
+import Card from '../../../Card';
+import {
+  blockchainSelectors,
+} from '../../../../redux/selectors';
+import stylesPage from '../../../../utils/pagesBase.module.scss';
+
+function CongressionalCountdown({ termDuration }) {
+  const currentBlockTimestamp = useSelector(blockchainSelectors.blockTimestamp);
+  const currentBlockNumber = useSelector(blockchainSelectors.blockNumber);
+  const remaining = currentBlockNumber % termDuration;
```

Comment:
this doesnt necessarily work if term duration changes, right ?

Reply:
It doesn't, but after a discussion I'm closing it